src/CodeGen_LLVM.h

```diff
@@ -332,17 +332,6 @@ class CodeGen_LLVM : public IRVisitor {
      * when multiple Allocate nodes shared the same memory. */
     virtual std::string get_allocation_name(const std::string &n) {return n;}
 
-    /** Helpers for implementing fast integer division. */
-    // @{
-    // Compute high_half(a*b) >> shr. Note that this is a shift in
```

Comment:
Did these comments get preserved somewhere? Otherwise it's hard to know what "sorted" means in sorted_avg.